dashboard/app/models/lesson.rb

```diff
@@ -41,6 +41,15 @@ class Lesson < ActiveRecord::Base
 
   serialized_attrs %w(
     overview
+    title
+    short_title
```

Comment:
title appears to be unused. also, what is the relationship between `short_title` and `I18n.t "data.script.name.#{script.name}.lessons.#{key}.name"`?

Reply:
also... our code is super inconsistent between `name`, `display_name`, `title`, and whether those are stored as table columns, properties, or just accessed via `I18n.t`. 

I would like to vote `display_name` off the island because we want to be able to use the "display" adjective to mean "processed in some way for showing to the user". For example, it can mean localized, plus other modifications sometimes made, such as when we add the unit number prefix here: https://github.com/code-dot-org/code-dot-org/blob/1e21440030e228afcce54177871367c512e74dd0/dashboard/app/models/script.rb#L1377

`name` also has a strike against it because it is sometimes the unique key, such as for unit_group, script and level.

so, what do you think about standardizing on `title` as the term for the untranslated value of a string that will appear as the name/title of something in the UI? then we can use `localized_title` or `title_for_display` if we need to translate/process it before displaying it in the UI.

any input @Hamms or @uponthesun ?

Reply:
Ya sorry title should be removed. I agree that we are not at all consistent on name/display_name/title. We should decide on one before adding activity, activity section, lesson and more

Reply:
I'm fine with using title if that works for everyone

Reply:
@davidsbailey Right now `name` is a column on the lesson table. Now that we have key I don't think we need it to be a column anymore and it could be a property and we could call it title. Is it worth doing the work now to change that?

Reply:
Also to answer your question about short_title. Short_title is a separate field that is used when we need a shortened version of the name of a lesson. Looking at it now Im wondering if we should leave out short title until we know when we will need it. Previously it was only used on the calendar and we are redesigning that

Reply:
`title` sounds ok to me.

When deciding whether to get rid of the `name` column, it'd be good to do a little analysis on whether anything depends on that being a column, but if not that sounds fine. I'd also be ok with making `title` an alias to `name` for Lesson for now.

Reply:
+1, I was about to give a different answer but Winter's is better! after aliasing `title` to the `name` column, you could also update any code to use `title` instead of name, and add a comment indicating that `name` is deprecated. that can all be done in 1 PR with any data migration.

My thinking is still a little fuzzy on when to use columns vs properties. Ideally we'd standardize there but I think it is probably more important that we pick good names for things and less important that we are super consistent about columns vs properties until we come up with a better policy there.

Reply:
Created follow up task